exampleCourse/infoCourse.json

```diff
@@ -37,10 +47,6 @@
         {"name": "nnytko2", "color": "turquoise1", "description": "Question originally written by nnytko2"},
         {"name": "astec", "color": "orange2", "description": "Question originally written by andrewstec"},
         {"name": "tpl101", "color": "gray1", "description": "Question originally written for the XC 101 example course"},
-        {"name": "fa17", "color": "gray1", "description": "Question written in Fall 2017"},
-        {"name": "sp18", "color": "gray1", "description": "Question written in Spring 2018"},
-        {"name": "su18", "color": "gray1", "description": "Question written in Summer 2018"},
-        {"name": "fa18", "color": "gray1", "description": "Question written in Fall 2018"},
         {"name": "v2", "color": "red1", "description": "v2 generation question"},
         {"name": "v3", "color": "green1", "description": "v3 generation question (freeform)"},
```

Comment:
Delete the `v2` and `v3` tags if we don't need them anymore?

Reply:
the "v3" tags are still used. I will have to remove them from questions, if we don't want to use this tag anymore. I can work on that.